clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -142,7 +142,8 @@ class ClioWebClient(
     val responseSource = Source
       .single(requestWithCreds)
       .via(connectionFlow)
-      .initialTimeout(requestTimeout)
+      .initialTimeout(responseTimeout)
+      .idleTimeout(responseTimeout)
```

Comment:
This still seems like the PR is making things more restrictive, not less. If you crank up the existing response timeout parameter to something huge, does that not fix it?

Reply:
It does not. The problem is the idle-timeout on the `Stream`